### PR TITLE
Niloo khorshid PR

### PR DESCRIPTION
**Pull request by nkhorshid**
Not sure, added new files?

### CODE REVIEW

File: Nilookh_1.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This is my new file
+A new line
```
Comment (PabloRdrRbl) [inline Nilookh_1.txt:2]: Hello Niloo, one cool feature, is that I can also add comments in the lines of code we want to discuss during the review process.

**Review by PabloRdrRbl — APPROVED** [review]
Review closed. Well done!

File: Nilookh_1.txt (hunk 1 above)
Comment (nkhorshid) [reply Nilookh_1.txt:2]: This @@ -0,0 + ... is something you added? How does this work?

Comment (PabloRdrRbl) [reply Nilookh_1.txt:2]: @@ -0,0 +1,2 @@

Those are the "coordinates" of the change. See for example this explanation: https://stackoverflow.com/questions/8558597/what-does-the-meta-line-with-at-signs-in-svn-diff-or-git-diff-mean

**Pull request merged**